pom.xml

```diff
@@ -1,74 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+    <groupId>org.example</groupId>
+    <artifactId>alvarium-sdk-java</artifactId>
+    <version>1.0-SNAPSHOT</version>
 
-  <groupId>com.alvarium.sdk</groupId>
-  <artifactId>alvarium-sdk</artifactId>
-  <version>1.0-SNAPSHOT</version>
+    <properties>
+        <maven.compiler.source>1.11</maven.compiler.source>
+        <maven.compiler.target>1.11</maven.compiler.target>
```

Comment:
The project does not compile using version `1.11` as target. replacing it with `11` worked for me.

Reply:
Updated to `11`